src/bot/cogs/cogmanager.py

```diff
@@ -0,0 +1,41 @@
+from discord.ext.commands import Cog, group, ExtensionNotFound, ExtensionAlreadyLoaded, NoEntryPointError, \
+    ExtensionFailed, ExtensionNotLoaded
+
+
+class CogManager(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.exceptions = (
+            ExtensionNotFound, ExtensionAlreadyLoaded, NoEntryPointError, ExtensionFailed, ExtensionNotLoaded)
+
+    @group()
+    async def cog(self, ctx):
```

Comment:
Can you please add the `Context` type from `discord.ext.commands` to every ctx parameter?